pkgs/by-name/wa/watchyourlan/package.nix

```diff
@@ -0,0 +1,24 @@
+{ pkgs, lib, ... }:
+let
+  inherit (pkgs) buildGoModule fetchFromGitHub;
+in
+buildGoModule rec {
+  pname = "watchyourlan";
+  version = "2.0.0";
+
+  vendorHash = "sha256-+7CMubbCrl+DsGmN9/2jCQ6zLDKvjJ6PdJx8iI1vKOQ=";
+
+  src = fetchFromGitHub {
+    owner = "aceberg";
+    repo = "WatchYourLAN";
+    rev = version;
+    hash = "sha256-Z4UitJKWZXGypNJ737DQNGuyDXIbyT7a1wOSc6N2s1k=";
+
+    meta = {
+      description = "Lightweight network IP scanner. Can be used to notify about new hosts and monitor host online/offline history.";
```

Comment:
```suggestion
      description = "Lightweight network IP scanner";
      longDescription = ''
        Lightweight network IP scanner. Can be used to notify about new hosts and monitor host online/offline history.
      '';
```

Reply:
Thanks, I've changed it to "Lightweight network IP scanner with web GUI" which is shorter, only one sentence, while still more informative, as it's not a cli scanner.